.travis.yml

```diff
@@ -106,6 +106,12 @@ matrix:
               false;
           fi
 
+    - <<: *basic-vm
+      name: "UTF-8 Check"
+      script:
+        # Make sure we're not introducing any non-UTF-8 character in text files
+        - git diff $TRAVIS_BRANCH...HEAD -U0 | ( grep -a '^+' || true ) | ( ! grep -axv '.*' )
```

Comment:
Is this check still valid or should we run `file -I <filename>` and check the encoding is one of the handful of acceptable options (at least ascii, and utf-8)

Reply:
From what I've read, `file -I` doesn't check the whole file and is therefore not always correct

Reply:
@Patater hopefully 15c947445d addresses your comment